CHANGELOG.md

```diff
@@ -3,6 +3,38 @@
 <!-- prettier-ignore-start -->
 
 
+## 0.7.21
+
+### Python
+
+- Restore the SQLite "fts4", "fts5", "rtree", and "geopoly" extensions on Mac/Linux
+
+See the
+[`python-build-standalone` release notes](https://github.com/astral-sh/python-build-standalone/releases/tag/20250712)
+for more details.
+
+### Enhancements
+
+- Add `--python-platform` to `uv sync` ([#14320](https://github.com/astral-sh/uv/pull/14320))
+- Add experimental `uv sync --output-format json` ([#13689](https://github.com/astral-sh/uv/pull/13689))
+- Support pre-releases in `uv version --bump` ([#13578](https://github.com/astral-sh/uv/pull/13578))
+- Add `-w` shorthand for `--with` in `uvx` and `uv tool run` ([#14530](https://github.com/astral-sh/uv/pull/14530))
+- Add an exception handler on Windows ([#14582](https://github.com/astral-sh/uv/pull/14582))
+- Add hint when Python downloads are disabled ([#14522](https://github.com/astral-sh/uv/pull/14522))
+- Add `UV_HTTP_RETRIES` to customize retry counts ([#14544](https://github.com/astral-sh/uv/pull/14544))
+
+### Bug fixes
+
+- Do not re-resolve with a new Python version in `uv tool` if it is incompatible with `--python` ([#14606](https://github.com/astral-sh/uv/pull/14606))
+- Fix handling of globs in `cache-key`: follow symlinks ([#13438](https://github.com/astral-sh/uv/pull/13438)) and `..` and improve performance ([#13469](https://github.com/astral-sh/uv/pull/13469))
```

Comment:
We might want to clarify the symlink behavior. We only resolve terminal symlinks in globs, I think? See https://github.com/astral-sh/uv/pull/13438#issuecomment-3059477520. I'd consider it an enhancement rather than a bug since this was not allowed by design before.

We might want to just split https://github.com/astral-sh/uv/pull/13469 into two notes and link to it twice. One enhancement to support parent paths in cache keys and another to improve the performance of cache keys.

Reply:
I think it could go either way. I'd lean toward enhancement personally, but only weakly so.